### ISSUE TITLE
[Client] Mekanism conflict

### ISSUE THREAD

**Issue by Ristovski** [WesCook/Nutrition#1]
Loads fine on server, but client-side causes a null pointer exception:

```
[20:27:03] [Client thread/ERROR] [FML]: Caught exception from Mekanism (mekanism)
java.lang.NullPointerException
	at ca.wescook.nutrition.events.EventPlayerAttachCapability.AttachCapabilitiesEvent(EventPlayerAttachCapability.java:17) ~[EventPlayerAttachCapability.class:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_287_EventPlayerAttachCapability_AttachCapabilitiesEvent_AttachCapabilitiesEvent.invoke(.dynamic) ~[?:?]
```
Mekanism could be at fault here too, not sure yet.

**Comment by erindal** [comment]
I can confirm this, full crash report: https://gist.github.com/erindalc/026fd6d5a9eba7bcc3afa3512d630432

**Comment by WesCook** [comment]
Fixed in v1.0.1.  Thanks for the report!
